src/undetected_chromedriver/__init__.py

```diff
@@ -17,7 +17,7 @@
 from __future__ import annotations
 
 
-__version__ = "3.4.6"
+__version__ = "3.2.1"
```

Comment:
Wrong version used?

Reply:
Yeah, I accidentally copied your patch thinking it was a patch on top of @DX37's pull request. I fixed it in my fork but now FlareSolverr isn't working so I need to figure out what happened.